seed/main/nv.xml

```diff
@@ -17,8 +17,8 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 		</orientation>
 	</layout>
 	<characters>
-		<exemplarCharacters references="R1">[a á ą {ą\u0301} b {ch} {ch’} d {dl} {dz} e é ę {ę\u0301} g {gh} h {hw} i í į {į\u0301} j k {kw} {kʼ} l ł m n o ó ǫ {ǫ\u0301} s {sh} t {tł} {tłʼ} {ts} {tsʼ} {tʼ} w x y z {zh} ʼ]</exemplarCharacters>
+		<exemplarCharacters references="R1">[a á ą {ą\u0301} b {ch} {ch‘} d {dl} {dz} e é ę {ę\u0301} g {gh} h {hw} i í į {į\u0301} j k {kw} {kʼ} l ł m n o ó ǫ {ǫ\u0301} s {sh} t {tł} {tłʼ} {ts} {tsʼ} {tʼ} w x y z {zh} ʼ]</exemplarCharacters>
```

Comment:
This seems wrong. You have changed {ch’} using U+2019 to  {ch‘} using U+2018, a different shape and not the one I see in Navajo orthography websites.

Also, all of the following use U+02BC, should they not all be consistent is using U+2019 (or U+2018 if that is really what is intended: ... {kʼ} ... {tłʼ} ... {tsʼ} {tʼ} ... ʼ]